Library/Homebrew/formula.rb

```diff
@@ -560,10 +560,10 @@ def installed_prefix
   end
 
   # The currently installed version for this formula. Will raise an exception
-  # if the formula is not installed.
+  # if the latest version is not installed.
   # @private
-  def installed_version
-    Keg.new(installed_prefix).version
+  def installed_version_if_latest
```

Comment:
Hmm, not huge on this name but can't think of anything better.

Reply:
Question: When would this method be preferred over using `latest_version_installed?` and `version`?

Reply:
When you specifically want it to fail/return `nil` if it's an outdated version. If there's minimal/no usage: it could be removed entirely.

Reply:
Once #8548 and Homebrew/homebrew-core#59988 are merged, `brew` and `homebrew-core` shouldn't be using `installed_version` anymore but it's possible that it's used in other taps

Reply:
In that case I reckon keep this name as-is and deprecate it so we can remove it eventually instead.